packages/react/src/components/Masthead/HeaderNavContainer.js

```diff
@@ -100,14 +151,16 @@ const HeaderNavContainer = ({ children }) => {
           className={`${prefix}--header__nav-caret-left`}
           aria-label="Masthead left caret"
           onClick={paginateLeft}
-          ref={caretLeftRef}>
+          ref={caretLeftRef}
+          hidden="true">
           <CaretLeft20 />
         </button>
         <button
           className={`${prefix}--header__nav-caret-right`}
           aria-label="Masthead right caret"
           onClick={paginateRight}
-          ref={caretRightRef}>
+          ref={caretRightRef}
+          hidden="true">
```

Comment:
Boolean attribute doesn't take a value:

```suggestion
          hidden>
```